cmd/glasskube/cmd/bootstrap_git.go

```diff
@@ -155,10 +205,49 @@ var bootstrapGitCmd = &cobra.Command{
 			}
 		}
 
-		fmt.Fprint(os.Stderr, doneMessage)
+		// try to open argo-cd every 5 seconds until it succeeds in order to check whether it is ready
+		fmt.Fprintf(os.Stderr, "\nglasskube argo-cd application applied successfully!\n\nWaiting for argo-cd to be ready...")
+		ctx, cancel := context.WithTimeout(ctx, 5*time.Minute)
+		defer cancel()
+
+		tick := time.NewTicker(5 * time.Second)
+		tickC := tick.C
+		defer tick.Stop()
+	WaitLoop:
+		for {
+			select {
+			case <-ctx.Done():
+				fmt.Fprintf(os.Stderr, "\nargo-cd has not become ready in the specified timeout.\n")
+				break WaitLoop
+			case <-tickC:
+				if res, err := open.NewSilentOpener().Open(ctx, argoCdPkg, "", "localhost", 0); err == nil {
+					res.Stop()
+					break WaitLoop
+				}
```

Comment:
I think a more lightweight approach could be extracting the code that determines the active pod in the `open` package to a separate function and using that instead. 

This could also be used as an easy solution for https://github.com/glasskube/glasskube/issues/1124. Once we implement the probes outlined in the comment below the above issue we can refactor the gitops bootstrap to use those as well